app/src/main/java/gholzrib/arctouchchallenge/ArcTouchChallengeApplication.java

```diff
@@ -0,0 +1,24 @@
+package gholzrib.arctouchchallenge;
+
+import android.app.Application;
+
+import okhttp3.OkHttpClient;
+
+/**
+ * Created by Gunther Ribak on 30/11/2016.
+ * For more information contact me
+ * through guntherhr@gmail.com
+ */
+
+public class ArcTouchChallengeApplication extends Application {
```

Comment:
Yes, it didn't had too. I did like this because it's the way I did in other projects, so it was kind of automatic to create the client on a Application class. I will remember this in future projects.